robot/basestation/static/css/themes/lofi.css

```diff
@@ -0,0 +1,232 @@
+/*
+Bootstrap Buttons (Purple): #445B7B
+Regular Buttons (Green): #3D7F7F
+Table (Red): #C34641
+Table (Light Grey): #EFEFEF
+Navbar (Salmon): #A38282
+Text (Off-White): #E4E1D1
+Background (Grey): #2B343B
+Button Hover (Blueish Light Grey): #BCD4DD
+*/
+
+/* Background color */
+html, body {
+  height: 100%;
+  background-color: #2B343B;
+}
+
+/* Navigation bar color */
+.navbar {
+  background-color: #A38282;
+}
+
+/* Whole lotta bootstrap dropdown menu styling */
+.dropdown .nav-link {
+  color:#445B7B; text-decoration: none;
+}
+
+.dropdown .btn {
+  background: #445B7B; color:#fff;
```

Comment:
CS (same line and lowercase letters for hex values)

Reply:
Fixed